TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/TimerViewModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using TimerCalculatorUtility.Enums;
+using TimerCalculatorUtility.Models;
+using Xamarin.Forms;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class TimerViewModel : INotifyPropertyChanged
+    {
```

Comment:
The same - create BaseViewModel which will implement INotifyPropertyChanged, then extend this class in your TimerViewModel and other view models.
We shouldn't repeat the same code in several places. It can be extracted to base view model.